examples/node-oauth/README-ngrok.md

```diff
@@ -0,0 +1,108 @@
+- [Folder Structure](#folder)
+- [Contributing](#contributing)
+- [License](#license)
+
+# ⚙️ Included Features <a name="features"></a>
+
+- [Miro REST API](https://developers.miro.com/docs/web-sdk-reference)
+  - [GET /boards/{board_id}](https://developers.miro.com/reference/get-specific-board)
+  - [POST /oauth/token](https://developers.miro.com/reference/exchange-authorization-code-with-access-token)
+
+# 🛠️ Tools and Technologies <a name="tools"></a>
+
+- [Express](https://expressjs.com/)
+- [Axios](https://axios-http.com/docs/intro)
+
+# ✅ Prerequisites <a name="prerequisites"></a>
+
+- Create a [new app in Miro](https://miro.com/app/settings/user-profile/apps)
+- You have a [Miro account](https://miro.com/signup/).
+- You're [signed in to Miro](https://miro.com/login/).
+- Your Miro account has a [Developer team](https://developers.miro.com/docs/create-a-developer-team).
+- Your development environment includes [Node.js 14.13](https://nodejs.org/en/download) or a later version.
+- All examples use `npm` as a package manager and `npx` as a package runner.
+- Install [ngrok](https://ngrok.com/).
+
+# 📖 Associated Developer Tutorial <a name="tutorial"></a>
+
+> To view a more in depth developer tutorial
+> of this topic see the [OAuth 2.0 Miro tutorial](https://developers.miro.com/docs/getting-started-with-oauth) on Miro's Developer documentation.
+
+# 🏃🏽‍♂️ Run the app with ngrok <a name="run"></a>
+
+1. Run `npm install` to install dependencies.
+
+2. Run `ngrok http 3000` and copy the forwarding address. It should look something like:
+
+`https://1ce8-45-94-204-74.ngrok-free.app`. This will be the value for `redirectURL` in your `.env` file.
+
+3. Rename the `.sample.env` file to `.env` and then fill in the values.
+
+> To find your client ID and secret, go to `Your apps`, and then select the app you just created to access its settings page. On the app settings page, go to `App Credentials`, and copy the app Client ID and Client secret values.
+> To find your `boardId` Go to your desired Miro board, copy the board ID from the URL, and paste it to your .env file boardId variable. For more info on how to get your board ID, see the [community question](https://community.miro.com/developer-platform-and-apis-57/where-can-i-find-board-id-3154).
+> The `redirectURL` is just the forwarding address from step 2. Once your `.env` file is completed, it should look similar to this:
+
+```
+clientID=12345678910
+clientSecret=12345678910abcdefg
+redirectURL=https://01-11-11-2011-74.ngrok-free.app
+boardId=abcdefghik=
+```
+
+4. Open the [app manifest editor](https://developers.miro.com/docs/manually-create-an-app#step-2-configure-your-app-in-miro) by clicking **Edit in Manifest**. \
+   In the app manifest editor, copy and paste the following yaml code:
+
+```yaml
+# See https://developers.miro.com/docs/app-manifest on how to use this
+appName: OAuth2.0 with NodeJS
+redirectUris:
+  - https://01-11-11-2011-74.ngrok-free.app
```

Comment:
```suggestion
  - https://[YOUR_NGROK_URL].ngrok-free.app
```

Reply:
Good to merge now? I've committed your suggestion :)